cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java

```diff
@@ -112,8 +112,7 @@ protected HBaseQueueAdmin(Configuration hConf,
     this.cConf = cConf;
     this.tableUtil = tableUtil;
     // todo: we have to do that because queues do not follow dataset semantic fully (yet)
-    String unqualifiedTableNamePrefix =
-      type == QUEUE ? QueueConstants.QUEUE_TABLE_PREFIX : QueueConstants.STREAM_TABLE_PREFIX;
+    String unqualifiedTableNamePrefix = type.toString();
```

Comment:
There is a separate effort right now to introduce a common abstract base for QueueAdmin (CDAP-1177), I am leaving it to there.